src/hb-ot-cmap-table.hh

```diff
@@ -1383,6 +1385,21 @@ struct EncodingRecord
   DEFINE_SIZE_STATIC (8);
 };
 
+static inline hb_codepoint_t
+_hb_remap_symbol_pua (hb_codepoint_t codepoint)
+{
+  if (codepoint <= 0x00FFu)
+  {
+    /* For symbol-encoded OpenType fonts, we duplicate the
+     * U+F000..F0FF range at U+0000..U+00FF.  That's what
+     * Windows seems to do, and that's hinted about at:
+     * https://docs.microsoft.com/en-us/typography/opentype/spec/recom
+     * under "Non-Standard (Symbol) Fonts". */
+    return 0xF000u + codepoint;
```

Comment:
While at it, can you change this (and comments etc) to use `os2.usFirstCharIndex`?